README.md

```diff
@@ -8,3 +8,88 @@
 # OVOS-config
 
 helper package to interact with mycroft config
+
+
+## Configuration Structure
+
+### ovos.conf
+The `ovos_config` package determines which config files to load based on `ovos.conf`.
+`get_ovos_config` will return default values that load `mycroft.conf` unless otherwise configured.
+
+`ovos.conf` files are loaded in the following order, with later files taking priority over earlier ones in the list:
+1. /etc/OpenVoiceOS/ovos.conf
+2. /etc/mycroft/ovos.conf (Deprecated)
+3. `XDG_CONFIG_DIRS` + /OpenVoiceOS/ovos.conf
+3. /etc/xdg/OpenVoiceOS/ovos.conf
+4. `XDG_CONFIG_HOME` (default ~/.config) + /OpenVoiceOS/ovos.conf
+
+A simple `ovos_config` should have a structure like:
+
+```yaml
+xdg: True
```

Comment:
this is deprecated and always true, it is ignored if present in config but always returned in utils for backwards compat, all references to xdg flag should be removed

Reply:
Ah, I thought it wasn't needed in the local config because its hard-coded there. I'll update the comment